src/main/webapp/src/components/PhoneDetails.js

```diff
@@ -42,6 +42,16 @@ class PhoneDetails extends React.Component {
         })
     };
 
+    addToCart = () => {
+        axios.post("/api/basket", this.state.phoneDetails).then( response => {
```

Comment:
why do you pass this.state.phoneDetails to the method? we should pass new object with id only

Reply:
+